metricbeat/metricbeat.reference.yml

```diff
@@ -507,7 +507,15 @@ metricbeat.modules:
   enabled: true
   period: 10s
   hosts: ["localhost:4949"]
-  node.namespace: node
```

Comment:
Should we mention this removal in the changelog?

Reply:
I removed this mention now that the refactor is so impactful. But I'll re-add something about that, yes :+1: